src/index.ts

```diff
@@ -42,18 +42,12 @@ export default function cssInjectedByJsPlugin({
             enforce: 'post',
             name: 'vite-plugin-css-injected-by-js',
             config(config, env) {
-                if (env.command === 'build') {
-                    if (!config.build) {
-                        config.build = {};
-                    }
-
-                    if (relativeCSSInjection == true) {
-                        if (config.build.cssCodeSplit == false) {
-                            config.build.cssCodeSplit = true;
-                            warnLog(
-                                `[vite-plugin-css-injected-by-js] Override of 'build.cssCodeSplit' option to true, it must be true when 'relativeCSSInjection' is enabled.`
-                            );
-                        }
+                if (env.command === 'serve') return
```

Comment:
This is not necessary as the modification must only be made if the command is 'build'